addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.NZD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.JPY" model="res.currency">
```

Comment:
This approach to activating only EUR and USD by default is very western-centric. So I am ok to reactive a few (2 or 3) common currencies when installing the Australian one. In a multi-currency configuration, it shouldn't be too hard to activate others as you get all when going to Accounting > Configuration > Currencies.

For the position, you can modify base so that other countries benefit from it too.